docs/ja/user-guide/api.md

```diff
@@ -31,15 +33,15 @@ APIのドキュメントは以下で見ることができます [api.screwdriver
 
 Swaggerのドキュメントは例とお試しのための編集可能なパラメータを含んでいます。`/v4/documentation`にアクセスし、APIを呼び出すための`Try it out!`ボタンをお試しください。 
 
-Swagger page:
-![Swagger page](../../../user-guide/assets/swagger-page.png)
+Swaggerページ:
+![Swagger page](../../../../user-guide/assets/swagger-page.png)
```

Comment:
This file path should be `(../../user-guide/assets/swagger-page.png)`

Reply:
Fixed with `./assets/〜` because the latest doc uses it
https://raw.githubusercontent.com/screwdriver-cd/guide/master/docs/user-guide/api.md

Reply:
`./assets/` does not exist yet.  we need creating `./assets` and moving image files to `./assets/〜`.

Reply:
Ah I get it.. It's because of the different path than default `en`

Reply:
Fixed!